jerry-core/jcontext/jcontext.h

```diff
@@ -103,6 +113,7 @@ typedef struct
   uint8_t valgrind_freya_mempool_request; /**< Tells whether a pool manager
                                            *   allocator request is in progress */
 #endif /* JERRY_VALGRIND_FREYA */
+  jerry_module_data_t module_data[JERRY_CONTEXT_MODULE_COUNT];
```

Comment:
My guess is we'll want to have a "feature-flag" (#define / #ifdef) around all of this so that people still have the option to go for the most bare bones config possible.